targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CPROTO_064_SB/device/COMPONENT_CM4/system_psoc6_cm4.c

```diff
@@ -171,7 +171,7 @@ void SystemInit(void)
 #if !defined(CY_IPC_DEFAULT_CFG_DISABLE)
 
 #ifdef __CM0P_PRESENT
-    #if (__CM0P_PRESENT == 0)
+    #if (__CM0P_PRESENT == 1)
```

Comment:
Don't we need to also update this for `TARGET_CY8CKIT_064S2_4343W`? We should also have Cypress review this change.

Reply:
Yes, cypress needs to review for `CY8CPROTO_064_SB`. But I'm unsure if this update is required for `TARGET_CY8CKIT_064S2_4343W`, @devran01 can you comment on this, please?

Reply:
@Patater @jainvikas8 We don't need this change in `TARGET_CY8CKIT_064S2_4343W`, as the mailbox semaphores will be initialised when mailbox is initialised rather than `systeminit()`.

Reply:
Why couldn't the mailbox semaphores for `TARGET_CY8CPROTO_064_SB` be initialized when the mailbox is initialized? Seems odd that `TARGET_CY8CPROTO_064_SB` or `TARGET_CY8CKIT_064S2_4343W` would be different in this regard.

Reply:
@jainvikas8 Why is this commit still present in the PR. Shouldn't this go away since the mailbox semaphores will be initialised when mailbox is initialised?

Reply:
@devran01 This change is required for `TARGET_CY8CPROTO_064_SB`, mailbox semaphores will not be initialized. Without this, the mbed-os blinky application fails.

Reply:
@jainvikas8 The patch https://github.com/Patater/trusted-firmware-m/commit/02a132b30992f25b9ea6d649250b35a3718d0f0d initialises mailbox semaphores during mailbox init. Therefore we don't need to have this commit.

Reply:
Fixed.